sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -3039,6 +3198,27 @@ public partial class RestorableSqlResourceData : Azure.ResourceManager.Models.Re
         public System.Collections.Generic.IReadOnlyList<string> CollectionNames { get { throw null; } }
         public string DatabaseName { get { throw null; } }
     }
+    public partial class RestorableTableGetResult : Azure.ResourceManager.Models.ResourceData
+    {
+        internal RestorableTableGetResult() { }
+        public Azure.ResourceManager.CosmosDB.Models.RestorableTablePropertiesResource Resource { get { throw null; } }
```

Comment:
Shall we rename this property to `Properties`?

Reply:
We already have this change in preview version and we are just making it stable. Also, we are following similar convention for existing APIs of sql and mongo so we decided to use same for these new ones as well to remain consistent.